app/views/subscriptions/_jobseeker_account_prompt.html.slim

```diff
@@ -0,0 +1,10 @@
+- if @jobseeker_account_exists
+  #sign-in-prompt
```

Comment:
i am not a fan of using id's like this i.e just as hooks for the tests which as far as i can tell is all this used for. in this day and age id's should only be used for anchor identifiers. possibly in some cases for JS hooks but now we have things like `getElementsByClassName` available you could argue a good case that we shouldnt use id's for this purpose either

Reply:
nothing to do with this PR but using id's in CSS is definatley an anti-pattern to unless you really really have to

Reply:
Would you prefer a class or data attribute?

Reply:
good question in that i have very much used data-attributes in the past for test hooks. so if i was going set a precedent for a consistent approach id say data attribute but we dont have a consistent approach. maybe we should. for now i dont mind class or attribute but not id, im saying in a very roundabout way

Reply:
in fact the class/data attribute is a part of the to page object/not to page object conversation we started

Reply:
Let me go data attribute for now pending page object spike